EIPS/eip-ens-as-holder

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
@SamWilsn what is going on here? 🤔

Reply:
I think it's required for the first draft to wait for the editor to assign a EIP number, hence I was adding this file without a number

Reply:
No, it was something else that was odd. It fixed itself though.

Reply:
https://github.com/ethereum/EIPs/commit/25d6c6b25893487fc0cd7a7837044502779d7dd7#diff-464c8e2f23e1608a3c2e5b5a2e153ce449fb69bebd257c649d3c5a095fd73155R1

^ this one?

Things get a little wacky if the preamble doesn't parse correctly in its entirety.